roles/openshift_service_catalog/tasks/install.yml

```diff
@@ -72,33 +72,49 @@
     state: list
   register: edit_yaml
 
+# only do this if we don't already have the updated role info
 - name: Generate apply template for clusterrole/edit
   template:
     src: sc_role_patching.j2
     dest: "{{ mktemp.stdout }}/edit_sc_patch.yml"
   vars:
     original_content: "{{ edit_yaml.results.results[0] | to_yaml }}"
+  when:
+  - '"servicecatalog.k8s.io" not in edit_yaml.results.results[0]| to_yaml'
+  - '"settings.k8s.io" not in edit_yaml.results.results[0]| to_yaml'
```

Comment:
would there be a way we could do a `oc policy can-i --as` to confirm that these permissions are available? or would we need a user to be set up first; that might be a better way to verify that these permissions are available yet